public/app/config/config.js

```diff
@@ -79,6 +79,11 @@ const VISIBILITIES = {
   PRIVATE: 'private',
 };
 
+const DEFAULT_FORMAT = 'v1';
+const ACTIONS_VERBS = {
+  SIGNOUT: 'signout',
```

Comment:
Can you confirm this matches what Graasp does (if it does anything)? Note @ahgentil, that this is for signing out of the whole application. I.e. equivalent to signing out of Graasp.

Reply:
The in the page view we're using:
```
  Login = 'login',
  Logout = 'logout',
```

Reply:
@ahgentil, do we do anything for the main Graasp?

Reply:
In graasp we have "logged in" and "logged out" - in the "old" actions collection.